cmd/servicelog/list.go

```diff
@@ -13,14 +13,6 @@ import (
 	"github.com/spf13/cobra"
 )
 
-const (
-	AllMessagesFlag      = "all-messages"
-	AllMessagesShortFlag = "A"
-	InternalFlag         = "internal"
-	InternalShortFlag    = "i"
-	ListclusterIDFlag    = "cluster-id"
-)
```

Comment:
I found this distracting/hard to flop back and forth from where the flags are defined so I removed them for ease of reading.